.palantir/revapi.yml

```diff
@@ -121,3 +121,11 @@ acceptedBreaks:
       old: "method java.util.concurrent.ExecutorService com.palantir.atlasdb.keyvalue.impl.AbstractKeyValueService::createThreadPoolWihtoutSpans(java.lang.String,\
         \ int, int) @ com.palantir.atlasdb.keyvalue.cassandra.CassandraKeyValueServiceImpl"
       justification: "Protected method"
+  "0.881.0":
+    com.palantir.atlasdb:atlasdb-cassandra:
+    - code: "java.method.parameterTypeChanged"
+      old: "parameter <T> long com.palantir.atlasdb.keyvalue.cassandra.thrift.ThriftObjectSizeUtils::getCollectionSize(java.util.Collection<T>,\
+        \ ===java.util.function.Function<T, java.lang.Long>===)"
+      new: "parameter <T> long com.palantir.atlasdb.keyvalue.cassandra.thrift.ThriftObjectSizeUtils::getCollectionSize(java.util.Collection<T>,\
+        \ ===java.util.function.ToLongFunction<T>===)"
```

Comment:
ABI break here should be fine, method is only used internally inside this class.

Reply:
Also, technically the arguments haven't changed either

Reply:
technically its a binary compatibility break that requires a recompile as `Function<T, java.lang.Long>` won't implicitly convert to `ToLongFunction<T>` at runtime, and in some cases could fail at compile time. I'm not concerned about making this change though as I don't see any uses of this outside atlasdb.

Reply:
womp :(

Reply:
arguably this method should be private or package protected